Exiled.Loader/Loader.cs

```diff
@@ -125,12 +128,22 @@ static Loader()
             .IgnoreUnmatchedProperties()
             .Build();
 
+        private static bool IsAdministrator => RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+            ? new WindowsPrincipal(WindowsIdentity.GetCurrent()).IsInRole(WindowsBuiltInRole.Administrator)
+            : geteuid() == 0;
+
         /// <summary>
         /// Runs the plugin manager, by loading all dependencies, plugins, configs and then enables all plugins.
         /// </summary>
         /// <param name="dependencies">The dependencies that could have been loaded by Exiled.Bootstrap.</param>
         public static void Run(Assembly[] dependencies = null)
         {
+            if (IsAdministrator)
```

Comment:
Since this is only used once, the check should be directly embedded inside the if statement.

Reply:
That, or as a local variable above it. It has no need to be a property.